### DISCUSSION TITLE
Rename `linear` to `lockupLinear` and `dynamic` to `lockupDynamic` function parameters

### DISCUSSION THREAD

**Discussion by PaulRBerg** [sablier-labs/v2-periphery#59] [Ideas]
It's one thing to use the "linear" and the "dynamic" shorthands in our tests, and quite another thing to use them in a public setting (i.e. the proxy target).

WDYT, @andreivladbrg?

https://github.com/sablierhq/v2-periphery/blob/b93fe2feb1687438405b67ed2323d354745cfa00/src/interfaces/ISablierV2ProxyTarget.sol#L71-L76

**Comment by andreivladbrg** [comment]
In my opinion, "linear" and "dynamic" are sufficiently descriptive, especially considering that their types are interfaces, which further clarify their names.

**Reply by PaulRBerg** [reply]
Good point regarding types, but try to consider the names as if it was the first time you heard about them - "linear" and "dynamic" sound like mathematical terms, not contract names. I will let this discussion open for a while to sleep on this topic, but in any case, we're not in a rush to decide this because we can always upgrade the proxy target if need be.

**Comment by PaulRBerg** [comment]
Implemented in https://github.com/sablier-labs/v2-periphery/pull/86.
